scalafix-tests/unit/src/main/scala/scalafix/test/DummyLinter.scala

```diff
@@ -5,7 +5,7 @@ import scalafix.LintMessage
 import scalafix.Rule
 import scalafix.RuleCtx
 
-object DummyLinter extends Rule("DummyLinter") {
+object DummyLinter extends Rule("DummyLinter", "DummyLinterDescription") {
```

Comment:
I think this is one example where it makes sense that descriptions are optional, requiring a description is quite cumbersome in some cases.